src/modules/fw_pos_control_l1/FixedwingPositionControl.cpp

```diff
@@ -1746,7 +1746,8 @@ FixedwingPositionControl::Run()
 			if (_control_mode.flag_control_offboard_enabled ||
 			    _control_mode.flag_control_position_enabled ||
 			    _control_mode.flag_control_velocity_enabled ||
-			    _control_mode.flag_control_acceleration_enabled) {
+			    _control_mode.flag_control_acceleration_enabled ||
+			    _control_mode.flag_control_altitude_enabled) {
```

Comment:
@sfuhrer Nitpick, but maybe you can use flag_control_climb_rate_enabled. I think that's how it's used in other locations and in theory could run TECS in climb rate mode. What do you think? Otherwise good to go.

Reply:
I don't think it matters right now, but we should do a pass to make sure these are handled consistently and to even know where the holes are in offboard.